RadixWallet/Clients/OnLedgerEntitiesClient/Helpers/OnLedgerEntitiesClient+ComplexTypes.swift

```diff
@@ -44,11 +41,35 @@ extension OnLedgerEntitiesClient {
 			)
 		}()
 
+		return try await fungibleResourceBalance(
+			resource,
+			resourceAmount: .exact(.init(nominalAmount: amount)),
+			guarantee: guarantee,
+			networkID: networkID
+		)
+	}
+
+	func fungibleResourceBalance(
+		_ resource: OnLedgerEntity.Resource,
+		resourceAmount: ResourceAmount,
+		guarantee: TransactionGuarantee? = nil,
+		poolContributions: [some TrackedPoolInteraction] = [] as [TrackedPoolContribution],
+		validatorStakes: [TrackedValidatorStake] = [],
+		entities: InteractionReview.Sections.ResourcesInfo = [:],
+		resourceAssociatedDapps: InteractionReview.Sections.ResourceAssociatedDapps? = nil,
+		networkID: NetworkID,
+		defaultDepositGuarantee: Decimal192 = 1
+	) async throws -> KnownResourceBalance {
+		let resourceAddress = resource.resourceAddress
+
+		@Dependency(\.resourcesVisibilityClient) var resourcesVisibilityClient
+		let hiddenResources = try await resourcesVisibilityClient.getHidden()
```

Comment:
Interesting, why was this added?

Reply:
Not added, just moved from the older function